src/main/java/net/sf/jabref/external/MoveFileAction.java

```diff
@@ -143,23 +144,16 @@ public void actionPerformed(ActionEvent event) {
                     }
                     chosenFile = sb.toString();
                 } else {
-                    chosenFile = FileDialogs.getNewFile(frame, file, Collections.singletonList(extension),
-                            JFileChooser.SAVE_DIALOG, false);
+                    System.out.println("Entry extension " + extension);
+                    chosenFile = new NewFileDialogs(frame, file.getPath()).saveNewFile().toString();
```

Comment:
Below you create a `File newFile` from it, so probably no need to first convert it to a string